github-actions/feature-request/action.yml

```diff
@@ -38,32 +38,42 @@ inputs:
   insufficient-votes-label:
     description: 'Label to add when the `close-when-no-sufficient-votes` is set to false and there are no sufficient number of votes or comments from unique authors.'
     default: 'insufficient votes'
+  limit:
+    description: 'Limits the number of issues we are going to run the action over.'
+    default: -1
   close-comment:
     description: 'Comment to post when closing the issue.'
     default: >
       Thank you for submitting your feature request! Looks like during the polling
       process it didn't collect a sufficient number of votes to move to the next stage.
 
+
       We want to keep Angular rich and ergonomic and at the same time be mindful about
       its scope and learning journey. If you think your request could live outside
       Angular's scope, we'd encourage you to collaborate with the
       [community](https://angular.io/resources?category=community) on publishing it
-      as an open source [package](https://angular.io/guide/creating-libraries)!
+      as an open source [package](https://angular.io/guide/creating-libraries).
+
 
       You can find more details about the feature request process in our
-      [documentation](https://github.com/angular/angular/blob/automation-docs/docs/GITHUB_PROCESS.md#feature-request-process).
+      [documentation](https://github.com/angular/angular/blob/master/docs/GITHUB_PROCESS.md#feature-request-process).
```

Comment:
We should point to a specific sha so that if the file ever moves or changes, the link we put in a comment still works.

Reply:
Opening a PR to framework right now. Once we merge it, I suppose we can get the change in together with the workflow configuration.